tests/unit/runner.html

```diff
@@ -0,0 +1,50 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="../node_modules/mocha/mocha.css" />
+    <title>Test Runner</title>
+    <style>
+      /* The testing area is positioned offscreen. You can make it visible
+         during debugging so you can see your rendered components. */
+      #test-area {
+        position: absolute;
+        left: -10000px;
+        top: -10000px;
+        width: 1000px;
+        height: 1000px;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="test-area"></div>
+    <div id="mocha"></div>
+    <script src="../node_modules/source-map-support/browser-source-map-support.js"></script>
+    <script>
+      (function() {
+        /**
+         * Enable source maps if we're running on a server.
+         *
+         * If we're being served through the file protocol we're gonna
+         * throw CORS errors so don't enable them then.
+         */
+
+        var protocol = document.location.protocol;
+        if (protocol !== 'file:') {
+          sourceMapSupport.install();
+        }
+      })();
+    </script>
+    <script src="../node_modules/mocha/mocha.js"></script>
+    <script>
+      mocha.setup('bdd');
+    </script>
+    <script src="../build/tests.js"></script>
```

Comment:
The setup for the HTML runner is not in this PR. I suggest taking the runner out as well and add it in a future issue.